src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
 }
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: var(--color-for-main-text);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__text-code {
+    margin-bottom: 16px;
+    color: var(--color-for-secondary-text);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price-wrapper {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__price-text {
+    color: var(--color-for-secondary-text);
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    width: 100%;
+    padding: 12px;
+    border-radius: 5px;
+    background-color: var(--color-for-button);
+    color: #fff;
+    border: 0;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px; /* 114.286% */
+    text-transform: uppercase;
+
+    &:hover {
+      color: var(--color-for-button);
+      background-color: #fff;
+      border: 1px solid;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  box-sizing: border-box;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars.stars--1 > .stars__star:nth-child(-n +1),
+.stars.stars--2 > .stars__star:nth-child(-n + 2),
+.stars.stars--3 > .stars__star:nth-child(-n + 3),
+.stars.stars--4 > .stars__star:nth-child(-n + 4),
+.stars.stars--5 > .stars__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
```

Comment:
The same still not fixed, move these styles to separate file
<img width="810" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/eddf5b6a-dc6a-4b63-aecf-0108e32a0a16">